vllm/v1/core/sched/async_scheduler.py

```diff
@@ -15,15 +16,24 @@ def _update_after_schedule(
         scheduler_output: SchedulerOutput,
     ) -> None:
         super()._update_after_schedule(scheduler_output)
+        spec_decode_tokens = scheduler_output.scheduled_spec_decode_tokens
         for req_id in scheduler_output.num_scheduled_tokens:
             request = self.requests[req_id]
+            spec_tokens = len(spec_decode_tokens.get(req_id, []))
```

Comment:
Suggest renaming the var for clarity
```suggestion
            cur_num_spec_tokens = len(spec_decode_tokens.get(req_id, ()))
```

Reply:
ok， i will fix it